src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -3365,14 +3369,14 @@ def test_kernel_stub_usage():
 
     usage_msg = (
         "usage: genkernelstub [-h] [-o OUTFILE] [-api API] [-l] filename\n"
-        "genkernelstub: error: too few arguments")
+        )
```

Comment:
Python 3 is printing a different error message on the second line:
```
genkernelstub: error: the following arguments are required: filename
```
Rather than choose based on the Python version I thought it best to just check for the first line